modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/construct/RouteComponentObjectFactory.java

```diff
@@ -60,7 +60,8 @@ public Object doGetObject() throws Exception {
         model.getNestedComponents().stream()
             .filter(component -> component instanceof NestedChainModel)
             .findFirst()
-            .ifPresent(chain -> parameters.put(chain.getName(), new ProcessorChainValueResolver(nestedProcessors)));
+            .ifPresent(chain -> parameters.put(chain.getName(),
```

Comment:
superflous

Reply:
added the `muleContext` as constructor parameter, that's the reason of the line break